Source/PLATEAUEditor/Private/ExtentEditor/PLATEAUExtentGizmo.cpp

```diff
@@ -18,7 +18,9 @@ FPLATEAUExtentGizmo::FPLATEAUExtentGizmo()
 void FPLATEAUExtentGizmo::DrawHandle(int Index, FColor Color, const FSceneView* View, FPrimitiveDrawInterface* PDI, double CameraDistance) {
     SphereMaterial.Get()->SetVectorParameterValue(FName("Color"), Color);
 
-    double radius = CameraDistance / 20;
+    double zoomRate = 1.0 / (CameraDistance / 10000);
+    double radius = FMath::Min(CameraDistance / 20, 500);
```

Comment:
https://docs.unrealengine.com/5.1/en-US/epic-cplusplus-coding-standard-for-unreal-engine/
現状則ってない部分も多いですが基本的にUnreal EngineのCoding Standardに従うようにお願いいたします。
```suggestion
    double Radius = FMath::Min(CameraDistance / 20, 500);
```